pom.xml

```diff
@@ -762,14 +762,19 @@
       <dependency>
         <groupId>org.json4s</groupId>
         <artifactId>json4s-jackson_${scala.binary.version}</artifactId>
-        <version>3.5.3</version>
+        <version>3.6.5</version>
```

Comment:
Hi, @igreenfield . The following is the JIRA description. Since 3.6.6 is already out, could you describe the reason why we don't use `3.6.6` into the PR description?
> it will be very good to upgrade to newer version

Reply:
I will upgrade to latest version, I think it is better